app/assets/stylesheets/functions/_modular-scale.scss

```diff
@@ -0,0 +1,116 @@
+@charset "UTF-8";
```

Comment:
I think we should just remove Neat’s use of the `modular-scale`, instead of bringing it over from Bourbon. It’s hardly used currently and doesn’t really serve a meaningful purpose. It’s already been discussed here: https://github.com/thoughtbot/neat/issues/346